utils/db.ts

```diff
@@ -171,6 +171,112 @@ export async function getItemsSince(msAgo: number) {
   });
 }
 
+// Notification
+export interface Notification {
+  userId: string;
+  type: string;
+  userFromId: string;
+  userFromLogin: string;
+  originId: string;
+  originTitle: string;
+  // The below properties can be automatically generated upon item creation
+  id: string;
+  createdAt: Date;
+}
```

Comment:
```suggestion
export interface Notification {
  userId: string;
  text: string;
  url: string;
  // The below properties can be automatically generated upon item creation
  id: string;
  createdAt: Date;
}
```
Can we simplify the `Notification` interface? The text for the notification would be generated upon creation, and the notification row in `/account/notifications` would link back to where the notification was generated. E.g. link back to the item where a comment was made. This would make extending notification functionality in the future fairly easy. WDYT?

Reply:
Makes sense! Working on it!
I'd still leave a `title` or `type` field. Will probably be good for filtering in the future.